nerfstudio/viewer/server/viewer_utils.py

```diff
@@ -639,25 +639,23 @@ def _render_image_in_viewer(self, camera_object, graph: Model, is_training: bool
         check_thread = CheckThread(state=self)
         render_thread = RenderThread(state=self, graph=graph, camera_ray_bundle=camera_ray_bundle)
 
-        oom = False
         with TimeWriter(None, None, write=False) as vis_t:
             check_thread.start()
             render_thread.start()
             try:
                 render_thread.join()
                 check_thread.join()
             except IOChangeException:
-                pass
+                del camera_ray_bundle
+                torch.cuda.empty_cache()
```

Comment:
You found these changes to help?

Reply:
They don't seem to hurt. They clear memory once in awhile.